.github/workflows/deploy-lambda-endpoints.yml

```
@@ -32,30 +38,84 @@ jobs:
+      - name: Cache Docker layers
+        uses: actions/cache@v4
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ matrix.folder }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-${{ matrix.folder }}
+
+      - name: Build Lambda for ${{ matrix.folder }}
+        run: |
+          echo "Building Lambda in infra/aws/lambda/endpoint/${{ matrix.folder }}" && \
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)